roxctl/common/flags/config_file.go

```diff
@@ -0,0 +1,139 @@
+package flags
+
+import (
+	"os"
+
+	"github.com/ghodss/yaml"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/pointers"
+)
+
+type Instance struct {
+	ProfileName       string `json:"name"`
+	CaCertificatePath string `json:"caCertPath"`
+	ApiTokenFilePath  string `json:"apiTokenPath"`
```

Comment:
What about allowing inline CA cert and token? For the latter, I'd suggest an additional check for the 600 file permissions and ownership.
```suggestion
	CaCertificatePath string `json:"caCertPath,omitempty"`
	CaCertificate string `json:"caCert,omitempty"`
	ApiTokenFilePath  string `json:"apiTokenPath,omitempty"`
	ApiToken  string `json:"apiToken,omitempty"`
```